src/test/java/co/rsk/federate/FederatorSupportTest.java

```diff
@@ -1,26 +1,46 @@
 package co.rsk.federate;
 
+import co.rsk.bitcoinj.params.RegTestParams;
 import co.rsk.config.BridgeRegTestConstants;
+import co.rsk.core.RskAddress;
+import co.rsk.crypto.Keccak256;
 import co.rsk.federate.adapter.ThinConverter;
 import co.rsk.federate.config.TestSystemProperties;
+import co.rsk.federate.signing.utils.TestUtils;
 import co.rsk.peg.Bridge;
 import co.rsk.peg.BridgeMethods;
+import co.rsk.peg.Federation;
+import co.rsk.peg.FederationMember;
 import org.bitcoinj.core.*;
 import org.bouncycastle.util.encoders.Hex;
 import org.ethereum.core.Blockchain;
+import org.ethereum.util.ByteUtil;
 import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
 import org.junit.Test;
 import org.mockito.stubbing.Answer;
 
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
+import java.util.Optional;
 
 import static org.mockito.Mockito.*;
 
 public class FederatorSupportTest {
 
-    private NetworkParameters networkParameters = ThinConverter.toOriginalInstance(BridgeRegTestConstants.getInstance().getBtcParamsString());
+    private NetworkParameters networkParameters;
+
+    private co.rsk.bitcoinj.core.NetworkParameters params;
+    private Federation federation;
+
+    @Before
+    public void setUp() throws Exception {
+        networkParameters = ThinConverter.toOriginalInstance(BridgeRegTestConstants.getInstance().getBtcParamsString());
+        params = RegTestParams.get();
+        federation = TestUtils.createFederation(params, 1);
```

Comment:
```suggestion
        federation = TestUtils.createFederation(RegTestParams.get(), 1);
```
No need to create the class variable `params` if it's only used here

Reply:
It is being used in more places. For ex:


![image](https://user-images.githubusercontent.com/94651730/191086428-b49ec0ad-eac7-4607-b8fa-03472306b283.png)

Reply:
I see, thanks